recipes/rad_haplotyper/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: rad_haplotyper
+  version: '1.1.6'
+
+source:
+  fn: v1.1.6.tar.gz
+  url: https://github.com/chollenbeck/rad_haplotyper/archive/v1.1.6.tar.gz
+  sha256: da7f663922ca66f4b736775df16e70c423dad74aeb03bc4d732572ff01d8466d
+
+build:
+  number: 0
+  skip: True #This program is a post analysis utility for use with the dDocent bioinformatics package, which has dependencies that will not compile on OS X.  Skipping to avoid confusion
```

Comment:
hehe, now it'll build on osx as well. 

Just put the `# [linux]` back and a `# needs ddocent, only available on linux` before or after that line